Metalama.Framework.Engine/CompileTime/ReferenceAssemblyLocator.cs

```diff
@@ -33,6 +33,8 @@ internal sealed class ReferenceAssemblyLocator
     {
         private const string _compileTimeFrameworkAssemblyName = "Metalama.Framework";
         private const string _defaultCompileTimeTargetFrameworks = "netstandard2.0;net6.0;net48";
+        private static readonly ImmutableArray<string> _defaultNugetSources =
+            ImmutableArray.Create( "https://api.nuget.org/v3/index.json", @"C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\" );
```

Comment:
Is it necessary to exclude the default sources? Aren't there other default sources on other platforms?

Reply:
You should not hardcode `C:\Program Files`. Make it at least using an environment variable on Windows.